ts/parts/Axis.ts

```diff
@@ -3590,6 +3590,33 @@ class Axis {
          */
         stackLabels: {
 
+            /**
+             * The animation configuration object for the stackLabels. Please
+             * note that this option only applies to the initial animation.
+             * For other animations, see [chart.animation](#chart.animation)]
+             * and the animation parameter under the API methods.
+             * The following properties are supported:
+             *
+             * - `defer`: The animation delay time in milliseconds.
+             *
+             * @sample {highcharts} highcharts/plotoptions/animation-ddefer/
+             *          Animation defer settings
+             *
+             * @type {Highcharts.AnimationOptionsObject}
+             */
+
+            animation: {
+                /** @internal */
+                defer: true
+            },
+
+            /**
+             * @default   true
+             * @type      {boolean|number}
+             * @since     next
+             * @apioption yAxis.stackLabels.animation.defer
```

Comment:
Why sometimes `defer` is defined as `boolean|number` (annotation, dataLabels, stackLabels) and for series is just `number` ?

Reply:
I think that setting `defer` as a boolean in series doesn't make sense, because setting it to true will set it as `series.animation.defer = 1`, meanwhile setting defer to `true` for others (annotation, dataLabels, stackLabels) makes that defer value is taken from this one set in series.